corehq/project_limits/models.py

```diff
@@ -36,3 +36,8 @@ class RateLimitedTwoFactorLog(models.Model):
     window = models.CharField(max_length=15, null=False)
     # 'number_rate_limited' is 19 characters, 31 for headroom
     status = models.CharField(max_length=31, null=False)
+
+    def save(self, *args, **kwargs):
+        kwargs['window'] = kwargs['window'] or 'unknown'
+        kwargs['status'] = kwargs['status'] or 'unknown'
+        super(RateLimitedTwoFactorLog, self).save(*args, **kwargs)
```

Comment:
we offlined, but this was a misunderstanding; this wasn't what I meant.

I meant
```
RateLimitedTwoFactorLog.objects.create(
    ip_address=ip_address,
    phone_number=number,
    username=username,
    method=method or 'unknown',
    status=status or 'unknown',
)
```
which is more explicit / less surprising, and also makes it clear that the point there is to make it say 'unknown' in the log if it's falsy

Reply:
change has been made 👍